pkg/tsdb/prometheus/models/query_test.go

```diff
@@ -140,12 +141,14 @@ func TestParse(t *testing.T) {
 			"expr": "rate(ALERTS{job=\"test\" [$__interval]})",
 			"format": "time_series",
 			"intervalFactor": 1,
+			"intervalMs": 60000,
 			"refId": "A"
 		}`, timeRange)
 
 		res, err := models.Parse(q, "15s", intervalCalculator, false)
 		require.NoError(t, err)
-		require.Equal(t, "rate(ALERTS{job=\"test\" [2m]})", res.Expr)
+		require.Equal(t, "rate(ALERTS{job=\"test\" [1m]})", res.Expr)
+		require.Equal(t, 120*time.Second, res.Step)
```

Comment:
Why are these different? We are not specifying `min step`, there be a same value for `$__interval` and step?

Reply:
It is because we are not providing the `maxDataPoints`, so it uses the safe one, ' 1100`. If you calculate with that you get that difference.

Reply:
👍 as we are not changing this in this PR, but I don't think Prometheus should be using `maxDataPoints` ad `Interval` and `IntervalMS` is already calculated using `maxDataPoints`.